features/schedules.feature

```diff
@@ -15,7 +15,7 @@ Feature: Schedule model
   @javascript
   Scenario: User visits schedule page and creates a schedule
     When I click "New Schedule"
-    And I fill in "date-input" with 1st of Jan, 2018
+    And I fill in "date-input" with Jan, 2018
```

Comment:
Why did we take away checking for a specific date?

Reply:
Because a planning period is for a given month. Not a specific date.